addons/mass_mailing/static/src/js/wysiwyg.js

```diff
@@ -20,6 +20,29 @@ const MassMailingWysiwyg = Wysiwyg.extend({
         return res;
     },
 
+    /**
+     * @override
+     */
+     getValue: function () {
+        let editable = this._super(...arguments);
+        const parsedDocument = new DOMParser().parseFromString(editable, "text/html");
+        const dropableZone = parsedDocument.querySelector('.o_editable');
+
```

Comment:
I understand we might sometimes want to add empty lines to dense functions, but here it's like four lines so I don't think it really adds any value.